libraries/Microsoft.Bot.Builder.Ai/SmartTalk.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Net.Http;
+using Newtonsoft.Json;
+using System.Threading.Tasks;
+using System.Linq;
+
+namespace Microsoft.Bot.Builder.Ai
+{
+    public class SmartTalk
```

Comment:
>SmartTalk [](start = 17, length = 9)

should this SmartTalkService class be in a library shareable by the V3 and V4 SDKs?